tests/test_credentials.py

```diff
@@ -81,7 +81,7 @@ class TestApplicationDefault(object):
 
     @pytest.mark.parametrize('app_default', [testutils.resource_filename('service_account.json')],
                              indirect=True)
-    def test_init(self, app_default): # pylint: disable=unused-argument
+    def test_init(self, app_default):
```

Comment:
You can suppress these warnings without needing to disable this check for all methods in tests:

For example, here you can
```python
def test_init(self, app_default):
    del app_default
```
If that's just too annoying, carry on.